### DISCUSSION TITLE
레벨3 방학 워크샵 계획

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#986] [General]
# 사용자 스토리 워크샵
## 사전 준비
- 코치들이 목표, 핵심 결과, 이니셔티브를 정하여 공유한다.
- 그룹을 할당한다.
    - 레벨3과 같은 방법
- 이니셔티브를 할당 받은 사람들끼리 기획 논의를 한다.
    - 이 작업이 목표와 핵심 결과를 달성하기에 적절한지
    - 적절하지 못하면 어떤 방향으로 접근해야할지

## 진행 순서 및 내용
- 필요한 기능들을 브레인 스토밍 한다
- 중복을 제거하고 유사한 것을 그룹핑하여 기능 목록을 준비한다
- 기능 목록의 우선순위를 정한다
- 기능의 사용자 스토리를 도출한다
- 사용자 스토리를 통해 맵을 만들고 스프린트와 배포 주기를 정한다
- 스토리별 담당자를 정한다

**Comment by gracefulBrown** [comment]
## 참고 자료
- [사용자 스토리 워크샵](https://hsboee.medium.com/user-story-workshop-f75e43e79c72)
- [사용자 스토리 워크샵-개선](https://hsboee.medium.com/user-story-workshop-%EA%B0%9C%EC%84%A0%EC%95%88-fe7fe09c0311)
- [사용자 스토리 도출](https://engineering-skcc.github.io/agile-leadership/Agile-Coaching-Experience-03/)

**Comment by gracefulBrown** [comment]
## 사용자 스토리 워크샵 방법

- 각 스쿼드의 목표와 이니셔티브 확인
- 이니셔티브를 위한 세부 작업 및 피처 도출
    - 포스트잇에 피처 작성
    - ex) 댓글 작성 날짜별 조회
- 중복 피처 제거 및 우선 순위 도출
- 피처에 대한 사용자 스토리 작성
    - 포스트잇에 사용자 스토리 작성
    - 포맷: 나는 {역할}로서 {이유}을 위해 {무엇}이 필요하다
    - 예시) 나는 **관리자**로서 **댓글 기능의 사용 패턴을 분석**하기 위해 **날짜별 댓글 작성 빈도수 정보**가 필요하다
- 도출된 사용자 스토리 기반으로 시나리오 도출
    - 사용자 스토리 맵을 활용
    - 시나리오는 하나의 테스크가 됨

**Reply by pocojang** [reply]
9/5일 시행
